### ISSUE TITLE
Optix Prime renders incorrectly (black) on Titan V (Volta)..

### ISSUE THREAD

**Issue by oscarbg** [jbikker/lighthouse2#1]
Some things I found trying to fix:
*Included Optix prime DLL&lib version is 5.1 so I updated to 6.0:
```
//#pragma comment(lib, "../OptiX/lib64/optix_prime.1.lib" )
#pragma comment(lib, "../OptiX/lib64/optix_prime.6.0.0.lib" )
```
now shows correctly:
```
running on GPU: TITAN V (80 SMs, 12GB VRAM)
OptiX Version:[6.0.0] Branch:[rel6.0] Build Number:[25650775] CUDA Version:[10.1] 64-bit 2019-01-29
```
*changed .cuda.cu files device support from :
compute_50,sm_50;compute_75,sm_75
to:
compute_70,sm_70;compute_75,sm_75
*I found some :
```
#if __CUDA_ARCH__ > 700
#define THREADMASK	__activemask() // volta, turing
#else
#define THREADMASK	0xffffffff // pascal, kepler, fermi
#endif
```
and changed:
`#if __CUDA_ARCH__ > 700`
to:
`#if __CUDA_ARCH__ >= 700`
to include Volta..
still black screen..

Note I also cleaned .cu.ptx file in case..
I'm using NV VK dev driver from 418 branch and CUDA 10.1 Update 1 SDK..
maybe fixed with original CUDA 10.1 SDK and/or NV 430.xx driver?
thanks..

**Comment by jbikker** [comment]
I found the problem.
While cleaning up the optixprime_b core for release I went a little too far. In pathtracer.cu you will find the shadeKernel function, and on line 55 the following statement:

const float4 T4 = pathLength == 1 ? make_float4( 1 ) : pathStateData[jobIndex * 2 + 0];	// path thoughput rgb 

Replace this by 

const float4 T4 = pathStateData[jobIndex * 2 + 0];	// path thoughput rgb 

to make the core work as intended.

Sorry for making you search for a solution!
